pom.xml

```diff
@@ -64,70 +60,6 @@
-                    <configuration>
-                        <skip>true</skip>
-                    </configuration>
-                </plugin>
-                <plugin>
-                    <groupId>org.eluder.coveralls</groupId>
-                    <artifactId>coveralls-maven-plugin</artifactId>
-                    <version>${coveralls-maven-plugin.version}</version>
-                    <configuration>
-                        <jacocoReports>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-common/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-jodatime/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-web/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-security-web/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-validator/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/target/site/jacoco/jacoco.xml</jacocoReport>
-                            <jacocoReport>terasoluna-gfw-common-libraries/terasoluna-gfw-string/target/site/jacoco/jacoco.xml</jacocoReport>
-                        </jacocoReports>
-                        <sourceDirectories>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-jodatime/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-security-web/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/src/main/java</sourceDirectory>
-                            <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-string/src/main/java</sourceDirectory>
-                        </sourceDirectories>
-                    </configuration>
-                </plugin>
-            </plugins>
-        </pluginManagement>
-    </build>
```

Comment:
I verified `deployAtEnd`. It was not effective.
I put `deployAtEnd` on `terasoluna-gfw` and I insert failure test case on purpose, then some artifacts were deployed in spite of occuring error.
And I put it on `terasoluna-gfw-common-libraries`, then all common-libraries were not deployed but `terasoluna-gfw-parent` was deployed.